extensions/amp-ad-network-doubleclick-impl/0.1/amp-ad-network-doubleclick-impl.js

```diff
@@ -1191,8 +1191,8 @@ export class AmpAdNetworkDoubleclickImpl extends AmpA4A {
    * @private
    */
   handleResize_(width, height) {
-    const pWidth = this.element.getAttribute('width');
-    const pHeight = this.element.getAttribute('height');
+    const pWidth = parseInt(this.element.getAttribute('width'), 10);
```

Comment:
I noticed L882 getSlotSize has dup logic . should we consolidate? (not sure if its fallback meets the goal here though).
in anyway, `Number` is more preferable than `parseInt` in this case.

Reply:
Yeah, I don't think the fallback works here, so best to keep separate. Switched to `Number`, though.

Reply:
I'd still encourage code sharing  by extracting a method just return the slot size. we might rename the current `getSlotSize` method to something else, as it involves such a fallback that does not seem relevant to its name.

Reply:
Factored out `getDeclaredSlotSize_` and updated some type declarations.